internal/service/ipam/api_subnet_resource_test.go

```diff
@@ -890,6 +890,16 @@ resource "bloxone_ipam_ip_space" "test" {
 `, name)
 }
 
+func testAccMultiSpace(space1, space2 string) string {
```

Comment:
might be better to rename as testAccBaseWithTwoIPSpace ? adding the base helps us know it isn't used in a test directly